portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/ExposeErcFromAssetLibrariesForVocabularies.testcase

```diff
@@ -159,6 +159,43 @@ definition {
 		}
 	}
 
+	@priority = "4"
+	test CanRecieveCorrectErcValueInCategory {
+		property portal.acceptance = "true";
+
+		task ("Given a TaxonomyVocabulary") {
+			var assetLibraryId = JSONGroupAPI._getDepotIdByName(depotName = "Test Depot Name");
+
+			var responseOfCreated = TaxonomyVocabularyAPI.createTaxonomyVocabulary(
+				assetLibraryId = "${assetLibraryId}",
+				externalReferenceCode = "erc",
```

Comment:
Is it needed to create the Taxonomy Vocabulary with erc? 🤔 It doesn't harm the test if you leave it, just asking out of curiosity if it is needed for this test. If it is, maybe it is worth mentioning it in the step

Reply:
I think don't need erc when creating the Taxonomy Vocabulary, maybe I need to modify it.